client/components/buttons/SurpriseButton.tsx

```diff
@@ -32,9 +33,13 @@ const SurpriseButton = () => {
       readingLevels,
       randomLocation,
       readingTimeOptions[randomReadingTime],
-      randomThemes.join(', ')
+      randomThemes.join(', '),
     );
 
+    const image_url = await createImage(readingLevels, randomLocation, randomThemes.join(', '));
+    const filename = `${storyDetails.id}.jpeg`;
+    await storeStoryImage(image_url, filename);
```

Comment:
this code is the same as in creating a new non-suprise story. wrap it around in a function